test-scripts/type-cframe.lua

```diff
@@ -0,0 +1,40 @@
+local function assertCFramePosition(vec, x, y, z)
+    assert(vec.X == x, ("%f ~= %f"):format(vec.X, x))
+    assert(vec.Y == y, ("%f ~= %f"):format(vec.Y, y))
+    assert(vec.Z == z, ("%f ~= %f"):format(vec.Z, z))
+end
+
+local function assertVector(vec, x, y, z)
+    assert(vec.X == x, ("x: %f ~= %f (%s)"):format(vec.X, x, tostring(vec)))
+    assert(vec.Y == y, ("y: %f ~= %f (%s)"):format(vec.Y, y, tostring(vec)))
+    assert(vec.Z == z, ("z: %f ~= %f (%s)"):format(vec.Z, z, tostring(vec)))
+end
+
+-- new with combinations of integer and floats
```

Comment:
FYI: I think it just means ints in the 5.3 (?) sense, where Lua added real integer types.

Reply:
Yeah I realized that cause I had to match on LuaValue::Integer + LuaValue::Number